docs/src/modules/components/Notifications.js

```diff
@@ -89,25 +126,70 @@ export default function Notifications() {
     return () => {
       active = false;
     };
-  }, [route]);
+  }, []);
 
   return (
-    <Snackbar
-      key={message.id}
-      anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
-      ContentProps={{ 'aria-describedby': 'notification-message' }}
-      message={
-        <span id="notification-message" dangerouslySetInnerHTML={{ __html: message.text }} />
-      }
-      action={
-        <Button size="small" color="secondary" onClick={handleClose}>
-          {t('close')}
-        </Button>
-      }
-      open={open}
-      autoHideDuration={20e3}
-      onClose={handleClose}
-      onExited={handleMessage}
-    />
+    <React.Fragment>
+      <Tooltip
+        open={tooltipOpen}
+        onOpen={handleTooltipOpen}
+        onClose={handleTooltipClose}
+        title={t('notifications')}
+        enterDelay={300}
+      >
+        <IconButton
+          color="inherit"
+          ref={anchorRef}
+          aria-controls={open ? 'notifications-popup' : undefined}
+          aria-haspopup="true"
+          aria-label={t('notifications')}
+          onClick={handleToggle}
+          data-ga-event-category="AppBar"
+          data-ga-event-action="notifications"
+        >
+          <Badge color="secondary" badgeContent={unseenNotificationsCount}>
+            <NotificationsIcon />
+          </Badge>
+        </IconButton>
+      </Tooltip>
+      <NoSsr>
+        <Popper
+          id="notifications-popup"
+          anchorEl={anchorRef.current}
+          open={open}
+          placement="bottom-end"
+          transition
+          disablePortal
+        >
```

Comment:
Popper has a default aria-role of "tooltip", which is a bit presumptuous of it! 😄 

Not sure what it should be instead though?

Reply:
No role seems fine here i.e. `role={null}`.

Reply:
Okay, thanks. Should I use `role="none presentation"` following https://www.w3.org/TR/wai-aria-1.1/#none?

Reply:
I don't have all the context at hand but "none presentation" wasn't all that useful. I think removing the role is the safest bet.